nav2_amcl/src/amcl_node.cpp

```diff
@@ -1390,6 +1391,10 @@ void
 AmclNode::mapReceived(const nav_msgs::msg::OccupancyGrid::SharedPtr msg)
 {
   RCLCPP_DEBUG(get_logger(), "AmclNode: A new map was received.");
+  if (!nav2_util::validateMsg(*msg)) {
```

Comment:
You can put this in the Static Layer's callback too :-)

Reply:
you mean such msg-type used in `costmap` ?